### PR TITLE
Upgrade prospero-metadata to 1.3.0.Beta2

### PR DESCRIPTION
**Pull request by TomasHofman**
This is the prospero-metadata version minor relevant to next EAP stream.

### CODE REVIEW

**Comment by TomasHofman** [comment]
This needs to be reviewed by @spyrkob .

**Review by spyrkob — APPROVED** [review]

File: plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java [hunk 1]
```diff
@@ -186,8 +186,7 @@ private void resolveFromChannels(MavenArtifact artifact) throws UnresolvedMavenA
 
     public void done(Path home) throws MavenUniverseException, IOException {
         ChannelManifest channelManifest = channelSession.getRecordedChannel();
-        final ManifestVersionRecord currentVersions = new ManifestVersionResolver(localCachePath, system)
-                .getCurrentVersions(channels);
+        final ManifestVersionRecord currentVersions = ManifestVersionResolver.getCurrentVersions(channelSession);
```
Comment (jamezp) [inline plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java:189]: Is there a reason for this change?

Comment (spyrkob) [reply plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java:189]: @jamezp I deprecated the old method - it was relying on a hack I want to get rid of. The new method uses changed wildfly-channel API to achieve the same goal more cleanly.

Comment (TomasHofman) [reply plugin/src/main/java/org/wildfly/plugin/provision/ChannelMavenArtifactRepositoryManager.java:189]: Yeah, that was really @spyrkob 's request.

**Pull request merged**